src/ansys/acp/core/_tree_objects/enums.py

```diff
@@ -56,7 +57,7 @@
     "DropOffType",
     "EdgeSetType",
     "ElementalDataType",
-    "ExtrusionType",
+    "ExtrusionGuideType" "ExtrusionType",
```

Comment:
```suggestion
    "ExtrusionGuideType",
    "ExtrusionType",
```
typo (missing comma)